repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/actor.py

```diff
@@ -264,6 +264,20 @@ def process(self):
             preparetransaction.remove_disk_image(mounts_dir)
             return
 
+        # secure yum upgradability (workaround)
+        syu_spath = preparetransaction.YUM_CFG_SCRIPT
+        syu_dpath = '/temp/'
+        preparetransaction.copy_file_to_container(ofs_info, syu_spath, syu_dpath)
+        cmd = [syu_dpath + 'copy_yum_v3_configuration.sh']
+        _unused, error = preparetransaction.guard_container_call(
+            ofs_info,
+            cmd,
+            guards=(preparetransaction.connection_guard(),)
+        )
+        if error:
+            error.summary = 'Could not secure yum upgradability.'
+            return error
+
```

Comment:
Only thing I can see is broken upgrade, even without properly produced  err message. There is not produced any error message... - and the `process` method should not return anything....